common/src/View/ViewPreferencePane.cpp

```diff
@@ -79,6 +81,18 @@ namespace TrenchBroom {
             setLayout(layout);
         }
 
+        namespace {
+            static constexpr int brightnessToUI(const float value) {
+                return static_cast<int>(vm::round(100.0f * (value - 1.0f)));
+            }
+
+            static constexpr float brightnessFromUI(const int value) {
+                return (static_cast<float>(value) / 100.0f) + 1.0f;
+            }
+
+            static_assert(0 == brightnessToUI(brightnessFromUI(0)));
```

Comment:
Should this rather be a test case?

Reply:
Not worth the trouble IMO or worth exposing these functions

Reply:
It seems weird to me to have this here. It seems like a very "random" thing to check. What about other values than `0`? What about `brightnessFromUI(brightnessToUI(0))`?

I'd remove this or turn it into a test case if it seems so important to you that it needs testing.

Reply:
I don't see why this is a problem, I just wanted to state a precondition. Code is worse if it's removed, so I don't want to remove it.

Reply:
Ok!